docs/ref/contrib/redirects.txt

```diff
@@ -14,28 +14,39 @@ Installation
 
 To install the redirects app, follow these steps:
 
-1. Ensure that the ``django.contrib.sites`` framework
-   :ref:`is installed <enabling-the-sites-framework>`.
-2. Add ``'django.contrib.redirects'`` to your :setting:`INSTALLED_APPS` setting.
-3. Add ``'django.contrib.redirects.middleware.RedirectFallbackMiddleware'``
+1. Add ``'django.contrib.redirects'`` to your :setting:`INSTALLED_APPS` setting.
+2. Add ``'django.contrib.redirects.middleware.RedirectFallbackMiddleware'``
    to your :setting:`MIDDLEWARE` setting.
-4. Run the command :djadmin:`manage.py migrate <migrate>`.
+3. Run the command :djadmin:`manage.py migrate <migrate>`.
+
+.. admonition:: Sites data migration
+
+    If you have existing redirects data with
```

Comment:
This is only true for a migration from an old version of Django. Therefore it should only be inside the release documentation  or in `versionchanged` note.

Reply:
Fair point, I'll update the docs, thanks.